File: ReactNative/src/modules/todo/services/create-todo.service.ts
import { ITodoForm } from '../../../@types/todo-form';
import { TokenAxios } from '../../common/utils/token-axios';
import { TODO_URL } from '../../common/utils/url';

export async function requestCreateTodo(form: ITodoForm): Promise<void> {
  const AXIOS = await TokenAxios();
  await AXIOS.post(TODO_URL, form);
}

File: flutter_todo/lib/src/modules/todo/services/create_todo_service.dart
import 'package:flutter_todo/src/modules/common/utils/token_dio.dart';
import 'package:flutter_todo/src/modules/common/utils/url.dart';

Future<void> requestCreateTodo(String memo) async => (await tokenDio())
    .post(todoUrl, data: {"memo": memo}).catchError((error) => throw error);
